README.md

```diff
@@ -111,6 +111,53 @@ $ curl -X POST -d '{"old_password":"dmytro.striletskyi.1337", "new_password":"dm
 | old_password | Input arguments validation | This field is required.                            | 400         |
 | new_password | Input arguments validation | This field is required.                            | 400         |
 
+* `POST | /user/password/recovery` - password recovery for an existing user by his email.
+
+##### Request parameters
+
+| Arguments | Type   | Required | Description  |
+| :-------: | :----: | :------: | ------------ |
+| email     | String | Yes      | User e-mail. |
+
+```bash
+$ curl -X POST -d '{"email":"dmytro.striletskyi@gmail.com"}' \
+      -H "Content-Type: application/json" \
+      http://localhost:8000/user/password/recovery | python -m json.tool
+{
+    "message": "Forgotten password has been sent to the specified e-mail address.",
```

Comment:
Outdated message. The message should say about recovery link.

Reply:
@dmytrostriletskyi, done.